### DISCUSSION TITLE
【求解】这个应用伪装模块注入TikTok的过程是否有问题？

### DISCUSSION THREAD

**Discussion by SekiBetu** [LSPosed/LSPosed#894] [Q&A] [answered]
[LSPosed_Verbose_2021-08-14T13_40_12.341.log](https://github.com/LSPosed/LSPosed/files/6985977/LSPosed_Verbose_2021-08-14T13_40_12.341.log)
[LSPosed_Verbose_2021-08-14T13_43_00.999.log](https://github.com/LSPosed/LSPosed/files/6985978/LSPosed_Verbose_2021-08-14T13_43_00.999.log)

第一个log是开机后点开tiktok，模块没有生效
第二个log是清空日志后单独点开tiktok，模块没有生效

**Comment by yujincheng08** [comment]
![image](https://user-images.githubusercontent.com/5022927/129438362-469164aa-23ff-41ab-ac15-7b489ca285d5.png)
这些应该就是应用伪装 hook 的。

**Comment by SekiBetu** [comment, marked as answer]
没问题了，我发现只是 hook TikTok 的时候不会弹出 hook 成功提示，hook 别的应用比如酷安的时候没问题，应该是 TikTok 搞了一些防 hook 的手段

**Reply by SekiBetu** [reply]
TikTok 也是牛的，新版本防 hook 了以后，老版本也会防 hook，大概是热更新了
